packages/EasyErrorHandler/src/Bridge/Laravel/Handler/Handler.php

```diff
@@ -0,0 +1,332 @@
+        $statusCode = $this->getResponseStatusCode($exception);
+        $headers = null;
+
+        $encodedPsrResponse = $encoder->encode($responseData, $statusCode, $headers);
+
+        return $this->createLaravelResponseFromPsr($encodedPsrResponse);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function renderForConsole($output, Exception $exception): void
+    {
+        (new ConsoleApplication())->renderThrowable($exception, $output);
+
+        $this->renderTranslationToConsoleIfNeeded($output, $exception);
+
+        $this->renderValidationFailuresToConsoleIfNeeded($output, $exception);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function report(Exception $exception): void
+    {
+        if ($this->shouldReport($exception) === false) {
+            return;
+        }
+
+        $logLevel = $exception instanceof LogLevelAwareExceptionInterface ?
+            $exception->getLogLevel() :
+            LoggerInterface::LEVEL_ERROR;
+
+        $this->logger->exception($exception, $logLevel);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function shouldReport(Exception $exception): bool
+    {
+        foreach ($this->dontReport as $type) {
+            if ($exception instanceof $type) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    /**
+     * Returns a determined user message.
+     */
+    protected function determineUserMessage(Exception $exception): string
+    {
+        $userMessage = null;
+        $userMessageParams = [];
+        if ($exception instanceof TranslatableExceptionInterface) {
+            $userMessage = $exception->getUserMessage();
+            $userMessageParams = $exception->getUserMessageParams();
+        }
+
+        return $this->translateMessage($userMessage ?? $this->defaultUserMessage, $userMessageParams);
+    }
+
+    /**
+     * Returns the current datetime string.
+     */
+    protected function getCurrentDateTimeString(): string
+    {
+        return Carbon::now()->toIso8601ZuluString();
+    }
+
+    /**
+     * Returns an exception converted into an array.
+     *
+     * @return mixed[]
+     */
+    protected function getResponseData(Exception $exception): array
+    {
+        return $this->useExtendedResponse() ?
+            $this->getExtendedResponseData($exception) :
+            $this->getShortResponseData($exception);
+    }
+
+    /**
+     * Returns a response status code based on an exception.
+     */
+    protected function getResponseStatusCode(Exception $exception): int
+    {
+        if ($exception instanceof StatusCodeAwareExceptionInterface === false) {
+            return 500;
+        }
+
+        return $exception->getStatusCode();
+    }
+
+    /**
+     * Checks whether the extended response should be used.
+     */
+    protected function useExtendedResponse(): bool
+    {
+        return $this->config->get('app.debug', false) === true;
```

Comment:
We should have a specific config value for that, to allow the application to use extended responses without turning debug on as it could trigger unexpected behaviours with other part of the app.

Reply:
done